.changelog/v0.14.0/summary.md

```diff
@@ -0,0 +1,17 @@
+*April 27th, 2022*
+
+This release notably features a new `query packet pending` command to 
+list outstanding packet commitments that are either unreceived or pending 
+acknowledgement at both ends of a channel.
+
+The `ibc` crate now also come with a complete [ICS 026][ics-26] implementation.
+
+### Note for operators
+
+The `create channel` command now requires an existing client and connection,
```

Comment:
@romac: Maybe let's also mention the new `query packet pending` command.

@mzabaluev will add the details of that to the guide. We only need to give a brief mention here.

Reply:
Done